README.md

```diff
@@ -19,11 +19,10 @@ We are using the following technologies in our project,
 4. Google Test
 5. Boost.Python
 
-Note that for building and installing currently only Linux systems are supported.
+Note that for building and installing for Linux systems following instructions should be followed. For windows [download](https://github.com/bits2zbytes/adaboost/raw/windowsSupportReadME/WINDOWS_SUPPORT_README.docx) this file and follow the given instructions.
```

Comment:
It would be good if something like following can be done,
```
Building from source
----------------------------

**Linux**

Set of instructions

**Windows**

Set of instructions

Reply:
I wasn't able to able to add pictures to the current readme, I added it but it didn't display. That's why I created another document.

Reply:
Pictures aren't necessary. You can avoid them.